public/error.html

```diff
@@ -41,8 +41,11 @@ <h2>This page isn't available</h2>
             </p>
             <a href="/" class="button router-link-active">Go to homepage</a>
             <p>
-              <small
-                >Please submit all bugs or issues to the project's GitHub issue repository (link in footer).</small>
+              <small>
+                Please submit all bugs or issues to
+                <a href="https://github.com/mozilla-iam/dino-park-issues">the
+                project's GitHub issue repository</a>.
+              </small>
             </p>
```

Comment:
From what I can see, this whole file isn't used anywhere - if you open people.mozilla.org/error.html directly, you'll see it's very broken - so I think we can just delete the whole file.